assignments/lambda-classes.js

```diff
@@ -1 +1,114 @@
 // CODE here for your Lambda Classes
+class Person {
+  constructor(attributes) {
+    this.gender = attributes.gender;
```

Comment:
More often than not, the variable name inside the constructor is `props`. There's no need to make them all unique in the future.